### PR TITLE
Added possibility to change border width of ComboBox widget.

### PR DESCRIPTION
**Pull request by kscd**
I added the possibility to change the border width of the ComboBox widget to an arbitrary integer. The default value is set to 1, the former standard value.

### CODE REVIEW

File: components/2.0/ComboBox.qml [hunk 1]
```diff
@@ -148,7 +151,7 @@ FocusScope {
         id: dropDown
         width: container.width; height: 0
         anchors.top: container.bottom
-        anchors.topMargin: -1
+        anchors.topMargin: 0
```
Comment (davidedmundson) [inline components/2.0/ComboBox.qml]: This seems unrelated to borders.

Seems somewhat sensible anyway; make sure you test that with a widget with border 1.

Comment (kscd) [reply components/2.0/ComboBox.qml]: I tested the change with a borderwidth of 1 and it looks how it should, for comparison I included a picture with a borderwidth of 5. The entries have smaller space because the overall height remains constant in both cases.

The 'anchors.topMargin: -1' prevented the line between the uppermost text and the list view from having a width of 2 pixel, but due to my code modifications this isn't an issue anymore.

borderwidth_5
borderwidth_1
